src/gui/src/displayControls.cpp

```diff
@@ -527,6 +527,10 @@ DisplayControls::DisplayControls(QWidget* parent)
   makeLeafItem(
       misc_.manufacturing_grid, "Manufacturing grid", misc, Qt::Unchecked);
   makeLeafItem(misc_.gcell_grid, "GCell grid", misc, Qt::Unchecked);
+  makeLeafItem(misc_.virtual_obstructions_and_blockages,
+               "Virtual Obstructions & Blockages (Debug)",
+               misc,
+               Qt::Unchecked);
```

Comment:
lets not add a debugging view to the main viewer, this should be a separate renderer